data/korath/korath hails.txt

```diff
@@ -0,0 +1,1470 @@
+		"Dewa Anda tidak dapat membantu Anda!"  # Your gods can't help you [now]!
+		"Hasrasa akis idradrpeb afampeb it kapihsa?"  # Will we sing together in the afterlife?
+		"Tevenref rapalafet rekubfi asra!"      # I mock your impotent rage!
+		"Rasosanef raipak raikafes!"            # [You will] do the dance of death!
+		"Kulit anda akan dilupakan."            # Your skin will be forgotten.
+		"Osa rasa kahafef ofraikafes."          # I will make a sculpture of your demise.
+		"Sara-sara rasagakperef ranicfapekes atra."  # [Our] children laugh at your [terrible piloting] skills.
+
+phrase "exile: I like to do things"
+	phrase
+		"exile: I like to"
+	word
+		" "
+	phrase
+		"exile: listen to (Band)"
+		"exile: eat good food"
+	word
+		.
+
+phrase "efreti: I like to do things"
+	phrase
+		"efreti: I like to"
+	word
+		" "
+	phrase
+		"efreti: listen to (Band)"
+		"efreti: eat good food"
+	word
+		.
+
+phrase "exile: I learned to do things"
+	phrase
+		"exile: Recently"
+	word
+		", "
+	phrase
+		"exile: i learned | practiced | taught"
+	word
+		" "
+	phrase
+		"exile: artistic or musical skill"
+		"exile: practical skill"
+	word
+		.
+
+phrase "efreti: I learned to do things"
+	phrase
+		"efreti: Recently"
+	word
+		", "
+	phrase
+		"efreti: i learned | practiced | taught"
+	word
+		" "
+	phrase
+		"efreti: artistic or musical skill"
+		"efreti: practical skill"
+	word
+		.
+
+phrase "exile: I like to"
+	word
+		"'aga tedarsoar"                            # I'm addicted to
+		"Tehapaniaar ahasi siahasitar samaf"        # Eternal bliss is enshrined in
+		"Teritfakar kelkirnni sabak sikefotar"      # Supreme enjoyment can be found in
+		"Kisat atar belrap asa dotob"               # There can never be enough
+		"Vicato 'emamo ferornno"                    # My soul always awaits
+		"Tomik 'aga feralit 'aga te"                # My skin draws me to
+		"Kerkorga 'aga asamap luki orkot ferduha"   # Surely I was bred to
+		"Hanaifara 'aga kisat ferduha"              # How can I not try
+		"'aga atar femeba'tar Nu'agel orkot"        # I'd give up Nutrigel for
+		"Helamip veri' temafir atar helr"           # Switching genders would be worth
+
+phrase "efreti: I like to"
+	word
+		"Mada sevartoar"                            # I'm addicted to
+		"Sebahaliaar abati tiabatisar tanaf"        # Eternal bliss is enshrined in
+		"Serisfakar kepkirlli tacak tikefosar"      # Supreme enjoyment can be found in
+		"Kitas asar ceprah ata vosoc"               # There can never be enough
+		"Wigaso menano ferorllo"                    # My soul always awaits
+		"Sonik mada ferapis mada se"                # My skin draws me to
+		"Kerkorda mada atanah puki orkos fervuba"   # Surely I was bred to
+		"Balaifara mada kitas fervuba"              # How can I not try
+		"Mada asar fenocasar Kuwaru hirlla"          # I'd forget the Quarg to
+		"Bepanih werim senafir asar bepr"           # Switching genders would be worth
+
+# Band names come from the human band list.
```

Comment:
No, because they're written in the Korath Exile language. Even if Endless Sky had an English to Indonesian translation built in, and a connection to the cipher, you still need some manual tweaking to get the words to look right.

Reply:
The `utils/korath-cipher.py` documents all of this. Although it is an executable script, most of the file is an explanation of the translation process, including the human component, which is most of the work. The script just automates the error-prone cipher.

Reply:
Yea, I was wondering about the creation of the band names and forgot to consider then translating it.